crates/meroctl/src/cli/context/identity.rs

```diff
@@ -25,21 +25,23 @@ pub struct ContextIdentityCommand {
 pub enum ContextIdentitySubcommand {
     #[command(about = "List identities in a context", alias = "ls")]
     List {
-        #[arg(help = "The context whose identities we're listing (omit to use default context)")]
-        #[clap(long = "context", short = 'c')]
-        context: Option<Alias<ContextId>>,
+        #[arg(
+            help = "The context whose identities we're listing (omit to use default context)",
```

Comment:
can you remove points of "(omit to use default context)" entirely?

it should already be in the help report, clap should already say what the default is